lib/phases/reboot.js

```diff
@@ -139,6 +151,21 @@ Reboot.prototype.testRun = function() {
     })
     .then(function() {
       return phase.waitForB2GStart();
+    })
+    .then(function() {
+      var promises = Promise.all([
+        phase.waitForMemory(Phase.VERTICAL_CONTEXT),
+        phase.waitForMemory(Phase.SYSTEM_CONTEXT)
+      ]);
+
+      phase
+        .triggerGC()
+        .then(function() {
```

Comment:
I notice that the --memoryDelay command line option is not implemented for when these memory values are gathered after reboot/restart. Is that intentional? Should the --memoryDelay be implemented here too, to be consistent? Or is there a reason you didn't want that option on reboot/restart?